knimin/lib/data_access.py

```diff
@@ -2570,6 +2570,7 @@ def delete_study(self, study_id):
         """
         self._study_exists(study_id)
         with TRN:
+            self._dissociate_all_samples_from_study(study_id)
```

Comment:
Is this safe? I have the feeling that we should not allow to remove the study if it has samples associated. I think of the study as a container of samples, and I have to have all the samples in a container. If I do this, it is possible that I end up with samples not associated to any study, which would be bad.